### DISCUSSION TITLE
Payment Gateway Button Issue

### DISCUSSION THREAD

**Discussion by jitendra00752** [duncanmcclean/simple-commerce#1017] [Help]
When i am entering dummy payment details, there is no submit button visible. Could you please provide assistance in understanding how it functions?

**Comment by duncanmcclean** [comment]
I've just spun up a fresh site using the Starter Kit and the submit button is showing for me:

<img width="652" alt="image" src="https://github.com/duncanmcclean/simple-commerce/assets/19637309/d4db8719-4b05-4b7b-ab2e-986c8780fe2b">

I *did* fix something on the payment view a week ago, maybe you still have a broken version in your project. Maybe try re-copying it into your project: https://github.com/duncanmcclean/sc-starter-kit/blob/main/resources/views/checkout/payment.antlers.html
